src/Uno.Extensions.Reactive/Operators/SelectAsyncFeed.cs

```diff
@@ -58,7 +58,7 @@ async void BeginEnumeration()
 							case OptionType.Some:
 								var previousProjection = projectionToken;
 								projectionToken = CancellationTokenSource.CreateLinkedTokenSource(ct);
-								projection = InvokeAsync(message, parentMsg, async ct2 => await _projection((TArg)data, ct2), context, projectionToken.Token);
+								projection = InvokeAsync(message, parentMsg, async ct2 => await _projection((TArg)data, ct2), null, context, projectionToken.Token);
```

Comment:
I wonder if we should used named parameters here, to make the null parameter more understandable.

Reply:
Yes, but actually it's just temporary here as I will add a parameter with the refresh PR